feat: 공통 UI 확장: Badge, Avatar, Loading, Dropdown, Toast(#13)

anneedesign-ui opened the pull request:
## 📋 작업 내용
- Badge 컴포넌트 구현
- Avatar 컴포넌트 구현
- Loading 컴포넌트 구현
- Dropdown Menu 컴포넌트 구현
- Toast (Sonner) 컴포넌트 구현

## ✅ 체크리스트
-  컴포넌트 구현 완료
-  로컬 테스트 완료
-  커밋 컨벤션 준수

Closes #13

anneedesign-ui committed:
feat: 공통 UI 확장: Badge(#10)

anneedesign-ui committed:
feat: 공통 UI 확장: Avatar(#10)

anneedesign-ui committed:
feat: 공통 UI 확장: Loading(#10)

anneedesign-ui committed:
feat: 공통 UI 확장: Dropdown(#10)

anneedesign-ui committed:
feat: 공통 UI 확장: Toast(#10)

anneedesign-ui committed:
feat: 공통 UI 확장: Badge, Avatar, Loading, Dropdown, Toast(#13)

anneedesign-ui committed:
fix: 컴포넌트 파일 정리 및 수정

G-jjunny commented on src/index.css:
```diff
@@ -0,0 +1,23 @@
+
+    /* OZ Gray Colors */
+    --oz-gray-white: 0 0% 98%;
+    --oz-gray-dark: 0 0% 20%;
```
해당 색상 변수는 소원님이 app.css에 작성해주셨어요!
관련 디자인 토큰은 한 곳에서 관리해주시면 좋을 것 같습니다.

G-jjunny commented on src/tailwind.config.ts:
tailwind v4에서는
기본 설정은 CSS에서 바로 가능해서 config 파일이 없어도 됩니다.

플러그인이나 고급 설정을 하면 다시 필요하지만 현재 색상/폰트 설정은 css에서 정의로 충분해 보입니다

G-jjunny commented on src/postcss.config.js:
해당 파일도 마찬가지에요 현재 환경에서는 postcss를 설치할 필요가 없습니다

anneedesign-ui committed:
fix: 중복 설정 파일 제거 및 색상 토큰 정리

- postcss.config.js 제거
- tailwind.config.ts 제거
- index.css 제거 (App.css에 이미 색상 정의 존재)

조교님 피드백 반영

anneedesign-ui committed:
fix: main.tsx에서 index.css import 제거
